cfgov/unprocessed/css/forms.less

```diff
@@ -4,179 +4,79 @@
+    padding: 0;
+    margin-top: 0.5em;
+    border: 1px solid @gray-50;
 
-    &.is-hovered,
-    &.is-focused {
-        z-index: 2;
-        border-color: @pacific;
-        outline: 1px solid @pacific;
-    }
-
-    &_icon {
-        // This element does not get added dynamically so it should be hidden
-        // until JavaScript is enabled.
-        display: none;
-        position: absolute;
-        top: 0;
-        right: 0;
-        z-index: 2;
-        min-width: 1em;
+    select {
+        height: 30px;
+        width: 100%;
         padding: unit( 4px / @base-font-size-px, em )
                  unit( 6px / @base-font-size-px, em );
-        // This will overlay .custom-select_text and make it appear as if any
-        // long words get cut off before they touch the icon.
-        border-left: 4px solid @white;
-        background: @gray-10;
-        color: @gray-50;
-        line-height: @base-line-height;
-        text-align: center;
-        .is-enabled & {
-            display: inline-block;
-        }
-    }
-
-    &_text {
-        position: absolute;
-        top: 0;
-        left: 0;
-        padding: unit( 4px / @base-font-size-px, em )
-                 unit( 10px / @base-font-size-px, em );
-        z-index: 1;
-        font-family: Arial, sans-serif;
-        line-height: @base-line-height;
-        color: @black;
-    }
-
-    &_placeholder {
-        color: @gray;
-    }
-
-    &_select {
-        position: relative;
-        // Push the native select arrow out of view
-        width: 100%;
-        max-height: unit((@base-line-height-px + (4px * 2)) / @base-font-size-px, em);
-        padding: unit( 4px / @base-font-size-px, em );
-        z-index: 3;
-        // Customize the colors
         border: 0;
-        border-radius: 0;
-        background-color: transparent;
-        background-image: none;
+        outline: 2px solid transparent;
+        appearance: none;
+        background-color: @white;
         color: @black;
-        -webkit-appearance: none;
-        box-shadow: none;
+        line-height: @base-line-height;
 
-        .is-hovered &,
-        .is-focused & {
-            outline: none;
+        &:hover, &:active, &:focus {
+            outline: 2px solid @pacific;
+            outline-offset: 0;
         }
     }
 
-    &__month {
-        max-width: unit(96px / @base-font-size-px, em);
+    select[disabled] {
+        color: @gray-80;
+        background-color: @gray-10;
+    }
+
+    select option:disabled {
+        color: @gray-80;
     }
 
-    &__year {
-        max-width: unit(86px / @base-font-size-px, em);
+    &:after {
+        position: absolute;
+        top: 0px;
+        right: 0px;
+        height: 30px; width: 30px;
+        border-left: 1px solid @gray-50;
+        background-color: @gray-10;
+        color: @gray-80;
+        content: @cf-icon-down;
+        font-family: 'CFPB Minicons';
+        line-height: 27px;
+        text-align: center;
+        pointer-events: none;
```

Comment:
This won't work on the latest IE, who are actually following the proper spec, so at some point it may not work elsewhere either. The reasoning behind the spec is to avoid nefarious links buried under other content. Not sure what the best way to move forward is, maybe a js click event?